File: demo-spring5-data-jpa-reactive/src/main/java/demo/MessageCommandLineRunner.java
package demo;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

import java.util.stream.LongStream;

@Component
class MessageCommandLineRunner implements CommandLineRunner {

    @Autowired
    private ReactiveMessageRepository reactiveMessageRepository;

    @Override
    public void run(String... strings) throws Exception {
        reactiveMessageRepository.deleteAll();
        LongStream.rangeClosed(1, 3).forEach(i -> reactiveMessageRepository.save(new Message(String.valueOf(i))));
        reactiveMessageRepository.findAll().forEach(System.out::println);
    }
}

File: demo-spring4-data-jpa-sync-kotlin/src/main/kotlin/demo/MessageCommandLineRunner.kt
package demo

import org.springframework.boot.CommandLineRunner
import org.springframework.stereotype.Component
import java.util.stream.Stream

@Component
class MessageCommandLineRunner(val messageRepository: SyncMessageRepository) : CommandLineRunner {

    override fun run(vararg args: String?) {
        messageRepository.deleteAll()
        Stream.of(Message(1, "1"), Message(2, "2"), Message(3, "3")).forEach { m -> messageRepository.save(m) }
        messageRepository.findAllAsStream().forEach { println(it) }
    }
}
